scripts/holiday-reminder.js

```diff
@@ -0,0 +1,66 @@
+const moment = require('moment-timezone');
+const holidays = require('@18f/us-federal-holidays');
+
+const TIMEZONE = process.env.TIMEZONE || 'America/New_York';
+const CHANNEL = process.env.HUBOT_HOLIDAY_REMINDER_CHANNEL || 'general';
+
+let lastRun = '';
```

Comment:
Should we use Hubot's brain ( https://ncapito.ghost.io/adding-a-brain-to-hubot/ ) for this? IIRC, Charlie is using Redis, though I can't remember where it's hosted. But the point is: if we end up needing to reboot Charlie a few times the day before a holiday, people are going to get a bit sick of it.

Reply:
As-coded, it will only report at _precisely 3:00 PM_.  I think the only risk of multiple reports is Charlie gets restarted after 3:00 PM and comes back up before 3:01 PM.  Really this is just flagging the last day that Charlie reported.

That said, there's no reason it couldn't go in the brain.  It'd take care of that one case!  I'll take care of it before the end of the week.